app/src/test/java/ca/klapstein/baudit/data/ProblemTest.java

```diff
@@ -13,8 +19,68 @@ public void setUp() {
     public void tearDown() {
     }
 
+
+    @Test
+    public void problemConstructor() {
+        Problem problem = new Problem("test_title", "");
+        assertNotNull(problem);
+        assertEquals("test_title", problem.getTitle());
+        assertEquals("",problem.getDescription());
+    }
+
+
+    @Test
+    public void testGetTitle(){
+        Problem problem = null;
+
+        try{
```

Comment:
instead of try catching you could add:

`throws IllegalArgumentException`

To the test methods signature.

e.g. 
```java
 public void testGetTitle() throws IllegalArgumentException {
     // test stuff 
}
```